src/index.html

```diff
@@ -1,17 +1,158 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class="group-field first-group">
+        <legend>Personal information</legend>
+        <label class="form-field first-input">
+          Surname:
+          <input
+            type="text"
+            autocomplete="off"
+            name="surname"
+          >
+        </label>
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            autocomplete="off"
+            name="name"
+          >
+        </label>
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="date"
+          >
+        </label>
+        <label class="form-field last-input">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="term"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="group-field">
+        <legend>Registration</legend>
+        <label class="form-field first-input">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label class="form-field last-input">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="8"
+            maxlength="32"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="group-field">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field first-input">
+          <label>
+            Do you love cats?
+            <input
+              type="radio"
+              name="cats"
+              value="yes"
+            >
+            Yes
+          </label>
+          <label>
+            <input
+            type="radio"
+            name="cats"
+            value="no"
+            >
```

Comment:
```suggestion
            <input
              type="radio"
              name="cats"
              value="no"
            >
```
check code style (everywhere) 🥲